apps/storybook/src/Breadcrumbs.stories.tsx

```diff
@@ -20,7 +20,7 @@ import {
 import {
   SvgChevronRightDouble,
   SvgFolder,
-  SvgMoreSmall,
+  SvgMore,
```

Comment:
Can we revert this change in breadcrumbs?

When the breadcrumbs were created in HTML/CSS first, I was using the `…` text symbol.  [I never noticed until recently](https://github.com/iTwin/iTwinUI/pull/1060/files#r1101599521) that Storybook had an example using `SvgMoreSmall`.  I would almost rather see breadcrumbs use the `…` text symbol rather than any SVG, but for sake of this keeping this PR scoped I would say just leave it as is.  You can close #1151 as well.
![What I would prefer to do rather than use an SVG](https://user-images.githubusercontent.com/849817/227018273-bbf7b1b0-28bf-4a58-a440-1c5e760fa2ac.png)

The difference I see between these 2 scenarios:
- Breadcrumbs is using `…` express there content that is being truncated, just like when setting `text-overflow: ellipsis;` in CSS.
- Tabs is using `SvgMore` as a symbol to show there is a menu, similar to in tiles component.  Tabs could very well be using a different looking symbol, it just so happens that we're using 3 dots to express this here as well.
![Screenshot 2023-03-22 at 3 44 08 PM](https://user-images.githubusercontent.com/849817/227018792-40920f06-2e2b-4475-a9ca-0bcd73b6c893.png)

Maybe the better thing for us to consider would be replacing `SvgMore` with some other symbol?  @VincentChabot1, something to keep an eye out for in other design systems for inspiration.

Reply:
Had a conversation with Leah about this icon.  The icon is not a default, it's only a sample, so leaving the icon change is fine.  Regarding #1151, I think we need to have a bigger discussion re: consistency.  We've added an item to our internal document for next major release planning to consider any changes here.